.dotnet/src/Generated/Internal/ModelSerializationExtensions.cs

```diff
@@ -8,14 +8,12 @@
 using System.Diagnostics;
 using System.Globalization;
 using System.Text.Json;
-using System.Xml;
 
 namespace OpenAI
 {
-    internal static class ModelSerializationExtensions
+    internal static partial class ModelSerializationExtensions
     {
         internal static readonly ModelReaderWriterOptions WireOptions = new ModelReaderWriterOptions("W");
-        internal static readonly BinaryData SentinelValue = BinaryData.FromObjectAsJson("__EMPTY__");
```

Comment:
I realize that this is a draft PR, so you can ignore this comment for now. Just wanted to add it as a reminder for me to follow up on this later.

There was a fix to change this from:

```csharp
internal static readonly BinaryData SentinelValue = BinaryData.FromObjectAsJson("__EMPTY__");
```

To:

```csharp
internal static readonly BinaryData SentinelValue = BinaryData.FromBytes("\"__EMPTY__\""u8.ToArray());
```

This was done to make it compatible with AOT compilation. But here I see this line is being deleted instead.

Reply:
Yep, this is captured in https://github.com/microsoft/typespec/issues/4753